apps/ledger-live-mobile/src/screens/Onboarding/steps/pairNew.tsx

```diff
@@ -116,7 +116,7 @@ function OnboardingStepPairNew() {
       });
       return;
     }
-    dispatch(completeOnboarding());
+    dispatch(completeOnboarding(true));
```

Comment:
can you check other calls of this method throughout the codebase and apply the same fix if needed?
For instance in `SyncOnboarding/index.tsx`